koku/reporting/provider/gcp/models.py

```diff
@@ -81,3 +101,137 @@ class Meta:
     cost_entry_product = models.ForeignKey(GCPCostEntryProductService, null=True, on_delete=models.CASCADE)
     cost_entry_bill = models.ForeignKey(GCPCostEntryBill, on_delete=models.CASCADE)
     project = models.ForeignKey(GCPProject, on_delete=models.CASCADE)
+
+
+class GCPCostEntryLineItemDaily(models.Model):
+    """GCP cost entry daily line item."""
+
+    class Meta:
+        """Meta for GCPCostEntryLineItem."""
+
+        db_table = "reporting_gcpcostentrylineitem_daily"
+        indexes = [
+            models.Index(fields=["usage_start"], name="gcp_usage_start_idx"),
+            GinIndex(fields=["tags"], name="gcp_cost_entry"),
+        ]
+
+    id = models.BigAutoField(primary_key=True)
+
+    cost_entry_bill = models.ForeignKey(GCPCostEntryBill, on_delete=models.CASCADE)
+    cost_entry_product = models.ForeignKey(GCPCostEntryProductService, null=True, on_delete=models.CASCADE)
+    project = models.ForeignKey(GCPProject, on_delete=models.CASCADE)
+
+    line_item_type = models.CharField(max_length=256, null=True)
+    usage_start = models.DateField(null=False)
+    usage_end = models.DateField(null=True)
+    tags = JSONField(null=True)
+    usage_type = models.CharField(max_length=50, null=True)
+    region = models.CharField(max_length=256, null=True, blank=True)
+    cost = models.DecimalField(max_digits=24, decimal_places=9, null=True, blank=True)
+    currency = models.CharField(max_length=256, null=True, blank=True)
+    conversion_rate = models.CharField(max_length=256, null=True, blank=True)
+    usage_amount = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    usage_unit = models.CharField(max_length=256, null=True, blank=True)
+    usage_to_pricing_units = models.DecimalField(max_digits=24, decimal_places=9, null=True)
```

Comment:
I think `usage_to_pricing_units` would make more sense as `usage_in_pricing_units` the to makes me think the column is a conversion factor and not the actual amount.